src/test/resources/fixtures/hmh/hmh-insighted/caliperGradeEventScore.described.json

```diff
@@ -33,7 +33,7 @@
         "attempt": "urn:uuid:{uuid is the same as object.id property, contained in this message}",
         "maxScore": 12,
         "scoreGiven": 6,
-        "scoredBy": "https://www.hmhco.com/AggPA"
+        "scoredBy": "urn:uuid:{uuid of the teacher who graded the assignment} | {software application that auto-graded the assignment}"
```

Comment:
up until now this grade event has been generated by a service and this service is identified by this scoredBy field. Similar to the above change, Ed Analytics implementation has chosen that this field be used by one Agent type, i.e. a service and not person.

If this event is being sent from the front end and not a service then it should be raised with Ed Analytics so that a discussion can happen on how we're meant to handle your grade events from both UI (front) and Service (back) sides

Reply:
The above change came about on Padraig's request.
Do we need to get together and clarify this?

Reply:
scoredBy can be a SoftwareApplication if the assessment was auto-graded, and a Person (the scorer who is likely the teacher) if it was manually scored